### ISSUE TITLE
[CHAOSS WEB]: Homepage - Hero section

### ISSUE THREAD

**Issue by yigakpoa** [chaoss/Accessibility#10]
### More on Guidelines

All content must be accessible using a keyboard as the only input method - [2.1.1: Keyboard](https://www.w3.org/WAI/WCAG21/quickref/#keyboard) 

### Please state the WCAG Level this issue falls under.

A

### Description

There is more than one way to locate a specific page in a website.

### Location

Homepage - Hero Section

### Actual Result

Our Homepage Hero section on the CHAOSS website has a: 

1. Broken same page link(target link doesn't exist/ does not have corresponding target).
2. information are not organized and makes keyboard usage difficult and confusing

### Your Evaluation

The links sequence has been broken and the screen reader announces information that are not recognized.

### Who is the best person to correct this?

Developer

### Who conducted this A11y test?

Victoria Ottah
